crates/adapters/src/controller/error.rs

```diff
@@ -85,7 +92,16 @@ impl Display for ConfigError {
     fn fmt(&self, f: &mut Formatter<'_>) -> Result<(), FmtError> {
         match self {
             Self::ConfigParseError { error } => {
-                write!(f, "Failed to parse pipeline configuration: '{error}'")
+                write!(f, "Failed to parse pipeline configuration: {error}")
+            }
+            Self::ParserConfigParseError {
+                endpoint_name,
+                error,
+            } => {
+                write!(
+                    f,
+                    "Error parsing format configuration for endpoint '{endpoint_name}': {error}"
```

Comment:
can we just show error here, and make endpoint_name an attribute that a user can look at?
(at least in UI context of `enpoint_name` is clear so it leads to unnecessary long error messages)

Reply:
This error gets thrown on pipeline startup. I suspect that right now, the UI will show the error message for the entire pipeline, without attributing it to a specific connector. If that's true, it's better to keep the endpoint name in the error message for now.